timemachine/fe/single_topology.py

```diff
@@ -1088,19 +1057,26 @@ def _setup_intermediate_chiral_bond_term(self, src_bond, dst_bond, lamb, interpo
 
         return potentials.ChiralBondRestraint(chiral_bond_idxs, chiral_bond_signs).bind(chiral_bond_params)
 
-    def setup_intermediate_state(
-        self,
-        lamb,
-        interpolate_harmonic_bond_params_fxn: Optional[ParameterInterpolationFxn] = None,
-        interpolate_harmonic_angle_params_fxn: Optional[ParameterInterpolationFxn] = None,
-        interpolate_periodic_torsion_params_fxn: Optional[ParameterInterpolationFxn] = None,
-        interpolate_nonbonded_params_fxn: Optional[ParameterInterpolationFxn] = None,
-        interpolate_chiral_atom_params_fxn: Optional[ParameterInterpolationFxn] = None,
-        interpolate_chiral_bond_params_fxn: Optional[ParameterInterpolationFxn] = None,
-    ):
+    def setup_intermediate_state(self, lamb, lambda_angles=0.4, lambda_torsions=0.7):
         """
         Setup intermediate states at some value of lambda.
+
+        Parameters
+        ----------
+        lamb: float
+
+        lambda_angles, lambda_torsions: float
+            For ring opening/closing transformations, alchemical parameter values controlling the intervals over which
+            bonds, angles, and torsions are interpolated. Note that these have no effect on terms not involved in ring
+            opening/closing.
+
+            - Bonds are interpolated in the interval [0, lambda_angles]
+            - Angles are interpolated in the interval [lambda_angles, lambda_torsions]
+            - Torsions are interpolated in the interval [lambda_torsions, 1]
         """
+
+        assert 0.0 < lambda_angles < lambda_torsions < 1.0
```

Comment:
nit: this is technically the open interval, but the docstrings seems to be the closed interval `<=`

Reply:
I might be misunderstanding the comment, but to me the current docstring doesn't indicate whether e.g. `lambda_angles` is allowed to be 0 or equal to `lambda_torsions`. Added a clarifying note in 25fd7aa

Reply:
I think this assertion is fine, I was confused about where exactly the interpolation starts:

```
- Bonds are interpolated in the interval [0, lambda_angles)
- Angles are interpolated in the interval [lambda_angles, lambda_torsions)
- Torsions are interpolated in the interval [lambda_torsions, 1]
```
- bond interpolation is when `0 <= lambda < lambda_angles`  
- angle interpolation is when   `lambda_angles <= lambda < lambda_torsions`
- torsion interpolation is when `lambda_torsions <= torsion <= 1`

(i.e. being clear about `<` vs `<=`)

Reply:
Oh, I see. I think the current docstring is correct in the usage of square brackets to indicate closed intervals (based on the implementation of `pad` [here](https://github.com/proteneer/timemachine/blob/fd37b16672994a4605a506c2f3380b6752c51aa8/timemachine/fe/interpolate.py#L175-L183)). I updated the docstring to use inequality notation to hopefully be less ambiguous in c3011a2.